registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/workflow/dto/SearchResponseDto.java

```diff
@@ -0,0 +1,42 @@
+package io.mosip.registration.processor.core.workflow.dto;
+
+import lombok.Data;
+
+@Data
+public class SearchResponseDto {
+
+	private static final long serialVersionUID = 6705845720255847210L;
+
+	/** The registration id. */
+	private String registrationId;
+
+	/** The status code. */
+	private String statusCode;
+
+	/** The status comment. */
+	private String statusComment;
+
```

Comment:
Can you include workflowType also here, it will be actual registrationType

Reply:
okay